src/main/java/com/telefonica/iot/cygnus/backends/mongo/MongoBackend.java

```diff
@@ -37,6 +45,12 @@
  */
 public class MongoBackend {
     
+    /**
+     * Available data models when persisting data at Mongo.
```

Comment:
I do not know if this comment is really referring to the Resolution enumeration. Is it? :)